### DISCUSSION TITLE
I forgot my strapi admin password in my dev environment and unable to recover it. Help!

### DISCUSSION THREAD

**Discussion by karthiks** [strapi/strapi#7751] [Legacy Topics]
Hi there,

How do I recover my forgotton password in my development environment. I have added some content to it manually lastnight and tonight I'm stuck in this issue.

You early help in this regard is much appreciated. I tried looking into `sendmail` docs but that didn't help. Strapi docs isn't of help here.

May be you should provide a dummy password username and hash, that we can use to rescue ourselves would help.

**Comment by derrickmehaffy** [comment]
@karthiks if you open your database in some kind of database client (DBeaver/HeidiSQL for SQL databases on all platforms, this includes SQLite which is the `--quickstart` default) you can delete the entries in the `strapi_administrators` table and when restarting Strapi it will ask you to register again.

**Reply by karthiks** [reply]
Thanks! This should go in Docs as re-setting Admin in non-prod environment for strapi. Should be a common need and might help others quickly.

**Reply by derrickmehaffy** [reply]
Normally you wouldn't have an issue but I know most ISPs block outbound port 25 (common insecure outbound SMTP port that sendmail uses) But we will be reviewing our docs over the next few months and I've added this to our list. We will be building out an alternative knowledge-base in the coming months for work arounds like this, and that is most likely where it will end up.
